corehq/apps/app_manager/templates/app_manager/partials/modules/case_list_multi_select.html

```diff
@@ -11,13 +19,29 @@ <h4 class="panel-title panel-title-nolink">
     </div>
     <div class="panel-body">
       <div class="row">
-        <div class="col-sm-8">
+        <div>
           <div class="checkbox">
             <label>
-              <input type="checkbox" data-bind="checked: multiSelectEnabled"/>
+              <input type="checkbox" id="multi-select-checkbox" data-bind="checked: multiSelectEnabled" onchange="reconcileCheckboxes()"/>
               {% trans "Enable multi-select case list" %}
             </label>
           </div>
+          <div class="checkbox">
+            <label>
+              <input type="checkbox" id="auto-select-checkbox" data-bind="checked: autoSelectEnabled, disable: !multiSelectEnabled()"/>
+              {% trans "Enable auto select" %}
+            </label>
+          </div>
+          <div data-bind="visible: multiSelectEnabled()">
+            <label class="control-label {% css_label_class %}">
+                {% trans "Maximum selected value" %}
+                <span class="hq-help-template" data-title="{% trans 'Maximum Selected Value' %}"
+                      data-content="{% trans_html_attr 'Sets a maximum selected value for multi select case lists. If no value is inputted, the default value is used.' %}">
```

Comment:
It'd be nice to say what the default value is, although it's a little bit annoying to need to pass the constant in here.

Although, if you did pass the constant into the template, you could also stick it in initial page data and then have javascript access the real value instead of having a magic number in two places.

Reply:
I tried this and had little luck with getting it passed in. I agree it would be best to add this so unless this is blocking I will fiddle with it more after this PR is merged

Reply:
not blocking